### PR TITLE
2.0.3_updates01

### PR DESCRIPTION
**Pull request by Kodalien**
@zhongjiajie 2.0.3  update  ZH-CN series 01

### CODE REVIEW

**Comment by zhongjiajie** [comment]
It a big change, I would take a look tonight when I have more time.

**Comment by Kodalien** [comment]
> 这是一个很大的变化，我今晚有空时会去看看。

@zhongjiajie  Hi brother ,I am refactoring all the way.

**Comment by Kodalien** [comment]
@zhongjiajie  why ?

**Comment by zhongjiajie** [comment]
@Kodalien So sorry about that, I forgot this issue, I would fetch it to local due to it is a big change

**Comment by zhongjiajie** [comment]
@Kodalien  This PR changed many files, and maybe next time we should separator to multiple PR. PR just like function in programing, one PR should do one single thing, and for easier to review

File: docs/zh-cn/2.0.3/user_doc/About_DolphinScheduler/About_DolphinScheduler.md [hunk 1]
```diff
@@ -1,10 +1,31 @@
...
+### 简单易用
+DAG监控界面，流程定义可视化，拖拽任务定制DAG，API方式与第三方系统对接, 一键部署
+### 高可靠性
+去中心化Master和Worker,采用任务队列避免过载，防止造成机器延时
+## 丰富的使用场景
+支持多任务类型、多租户
+## 高扩展性
+支持自定义任务类型，调度能力随集群线性增长，Master和Worker支持动态上下线
+
+## 组成
```
Comment (zhongjiajie) [inline docs/zh-cn/2.0.3/user_doc/About_DolphinScheduler/About_DolphinScheduler.md:15]: I do not think add DolphinScheduler composition here is a good point. IMO, most users could just ignore the composition of DS

**Review by zhongjiajie — COMMENTED** [review]
So sorry about that, this patch is too big to review, it contains multiple sections of the document, with format changed and content changed, Do you mind to separate to multiple PR?

File: docs/zh-cn/2.0.3/user_doc/About_DolphinScheduler/About_DolphinScheduler.md [hunk 2]
```diff
@@ -1,10 +1,31 @@
...
-支持自定义任务类型，调度器使用分布式调度，调度能力随集群线性增长，Master和Worker支持动态上下线
+
+# 原则
+
+### 简单易用
+DAG监控界面，流程定义可视化，拖拽任务定制DAG，API方式与第三方系统对接, 一键部署
+### 高可靠性
+去中心化Master和Worker,采用任务队列避免过载，防止造成机器延时
+## 丰富的使用场景
+支持多任务类型、多租户
```
[the remark this answers is not in the document]
Comment (Kodalien) [reply docs/zh-cn/2.0.3/user_doc/About_DolphinScheduler/About_DolphinScheduler.md:11]: transfer to other places @zhongjiajie

File: docs/zh-cn/2.0.3/user_doc/About_DolphinScheduler/About_DolphinScheduler.md (hunk 1 above)
Comment (Kodalien) [reply docs/zh-cn/2.0.3/user_doc/About_DolphinScheduler/About_DolphinScheduler.md:15]: overall reconstruction decision @zhongjiajie

File: docs/zh-cn/2.0.3/user_doc/About_DolphinScheduler/About_DolphinScheduler.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (zhongjiajie) [reply docs/zh-cn/2.0.3/user_doc/About_DolphinScheduler/About_DolphinScheduler.md:11]: Hi, @Kodalien  do you mind separating this PR into multiple and keeping it easy to review?

**Comment by Kodalien** [comment]
> 顺便说一句，我发现你只是更改了中文版文档，我们也应该更改英文版文档。

Is first ZH after  EH @zhongjiajie

**Comment by zhongjiajie** [comment]
Hi, @Kodalien do you mind separating this PR into multiple and keeping it easy to review?

**Comment by Kodalien** [comment]
> @Kodalien 这个 PR 更改了许多文件，也许下次我们应该分隔多个 PR。PR 就像编程中的函数一样，一个 PR 应该做一件事情，并且更容易查看

@zhongjiajie   CN <--> US 


> 嗨，@Kodalien，您介意将此 PR 分成多个并使其易于查看吗？

ok!! but will let you crash review @zhongjiajie

**Comment by zhongjiajie** [comment]
@Kodalien It does matter. Please just separate to multiple PR. thanks

**Pull request closed without merging**